### PR TITLE
DDS-1374 [Toggle] Retouching

### PR DESCRIPTION
**Pull request by poetrainy**
## Related Ticket(s) and Links

- JIRA Ticket: [DDS-1374](https://daikinsvoil.atlassian.net/browse/DDS-1374)

## Checklist

- [x] I have added a changeset file that describes the changes.
- [x] I have performed a self-review of my code.
- [x] I have added/updated appropriate regression tests.
- [x] I have confirmed that there are no accessibility warnings that can be addressed.
- [x] I have confirmed that the bug fix does not affect the keyboard accessibility of the component.
- [x] I have confirmed that the fix does not introduce any new visual regressions.

[DDS-1374]: https://daikinsvoil.atlassian.net/browse/DDS-1374?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/toggle/daikin-toggle.ts [hunk 1]
```diff
@@ -2,58 +2,41 @@ import { cva } from "class-variance-authority";
...
-
-type ToggleVariantProps = MergeVariantProps<typeof cvaToggle>;
+const TOGGLE_CLASS_NAME = cva([
+  "w-16",
+  "h-8",
+  "relative",
+  "bg-daikinNeutral-600",
+  "rounded-full",
+  "cursor-pointer",
+  "transition-[background]",
```
Comment (yodas7) [inline src/components/toggle/daikin-toggle.ts:13]: nit.

```suggestion
  "transition-colors",
```

File: src/components/toggle/stories/daikin-toggle.stories.ts [hunk 2]
```diff
@@ -37,7 +38,7 @@ export const Default: Story = {
     await expect(toggle).not.toBeChecked();
 
     // should react if inner toggle clicked
-    await step("Try to click toggle first time", async () => {
+    await step("Try to click toggle", async () => {
```
Comment (yodas7) [inline src/components/toggle/stories/daikin-toggle.stories.ts:41]:
```suggestion
    await step("Try to toggle by mouse click", async () => {
```

File: src/components/toggle/stories/daikin-toggle.stories.ts [hunk 3]
```diff
@@ -46,8 +47,9 @@ export const Default: Story = {
       await expect(toggle).toBeChecked();
     });
 
-    await step("Try to click toggle second time", async () => {
-      await userEvent.click(toggle);
+    await step("Try to press keyboard", async () => {
```
Comment (yodas7) [inline src/components/toggle/stories/daikin-toggle.stories.ts:50]:
```suggestion
    await step("Try to toggle by space key", async () => {
```

**Review by yodas7 — APPROVED** [review]
LGTM

**Review by hiro-daikin — APPROVED** [review]

**Pull request merged**